packages/block-library/src/image/edit.js

```diff
@@ -357,21 +357,7 @@ export class ImageEdit extends Component {
 			isEditing: false,
 		} );
 
-		const { id, url, alt, caption } = this.props.attributes;
```

Comment:
Should it be part of this PR? Doesn't seem to be relevant.

Reply:
Huh, no idea how that happened 🤔 Maybe an issue caused by a rebase I did before pushing this PR. Will revert.

Reply:
Reverted now.